python/python3-tekore/python3-tekore.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="python3-tekore"
-VERSION="4.3.0"
+VERSION="5.1.1"
 HOMEPAGE="https://pypi.org/project/tekore/"
-DOWNLOAD="https://files.pythonhosted.org/packages/60/35/faea7eb3e936eeb30edb196245ce280512139c084afd5245a6fe3cbaa477/tekore-4.3.0.tar.gz"
-MD5SUM="d58ba00cd9271d97c926b52b257a158e"
+DOWNLOAD="https://github.com/felix-hilden/tekore/archive/v5.1.1/tekore-5.1.1.tar.gz"
+MD5SUM="12250450c0584df39513439d5e8e52f5"
 DOWNLOAD_x86_64=""
 MD5SUM_x86_64=""
-REQUIRES=""
+REQUIRES="python3-setuptools-opt"
```

Comment:
You need to actually change your slackbuild to use the setuptools installed in /opt.

For example:

https://github.com/SlackBuildsOrg/slackbuilds/blob/ee8422d3d2d914051f468983ccf81c2dd474791d/python/python3-tempora/python3-tempora.SlackBuild#L73-L74

Reply:
Hello!
Let me try that. thanks.